### PR TITLE
Enable websockets and UI fixes

### PR DESCRIPTION
**Pull request by ddorgan**
Enable websockets
Remove duplicate ports in docker config file
Use v1.7.2 docker image until beta is pointed at this version. Then I'll revert this.
Fix ethstats

### CODE REVIEW

**Comment by ddorgan** [comment]
Peter, are you happy with the websocket and ethstats fixes?

File: config/docker/client.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
    lient:
        container_name: client
-       image: parity/parity:beta
```
Comment (keorn) [inline config/docker/client.yml]: I would leave it, if its broken then lets wait for the fix.

File: config/spec/authority_round.toml [hunk 2]
```diff
@@ -7,6 +7,9 @@ password = ["/parity/password"]
 [network]
 reserved_peers = "/parity/reserved_peers"
 
+[ui]
+force = true
+interface = "0.0.0.0"
```
Comment (keorn) [inline config/spec/authority_round.toml]: Why does ui have to be forced?
Cant the interface be just localhost?

**Comment by ddorgan** [comment]
I'm currently building a new beta docker image based on 1.7.2, so that issue should be resolved. Also you're right the UI does not need to be forced. However if it just listens on 127.0.0.1 then docker cannot forward to port, that's why I'm using 0.0.0.0.

**Comment by ddorgan** [comment]
parity/parity:beta docker image updated.

**Pull request merged**